test/unit/webdriver/device/keyboard_test.py

```diff
@@ -32,3 +32,65 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{keycode: 86}'
+        )
+        assert isinstance(driver.press_keycode(86), WebDriver)
+
+    @httpretty.activate
+    def test_keyevent(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/keyevent'),
+            body='{keycode: 86}'
+        )
+        assert isinstance(driver.keyevent(86), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode_with_flags(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{keycode: 86, metastate: 2_097_153, flags: 44}'
```

Comment:
`2_097_153` this style does not work in Python. Here should be `2097153`

Reply:
Ok, added.